packages/api-headless-cms/__tests__/environmentAliases.test.js

```diff
@@ -0,0 +1,97 @@
+import mdbid from "mdbid";
+import useGqlHandler from "./utils/useGqlHandler";
+import { Database } from "@commodo/fields-storage-nedb";
+import slugify from "slugify";
+
+const toSlug = text =>
+    slugify(text, {
+        replacement: "-",
+        lower: true,
+        remove: /[*#\?<>_\{\}\[\]+~.()'"!:;@]/g
+    });
+
+const fields = `
+    id
+    name
+    slug
+`;
+
+const READ_ENVIRONMENT_ALIAS = /* GraphQL */ `
+    query getEnvironmentAlias($id: ID!) {
+        cms {
+            environmentAlias: getEnvironmentAlias(id: $id){
+                data {
+                    ${fields}
+                }
+                error {
+                    code
+                    message
+                }
+            }
+        }
+    }
+`;
+
+const CREATE_ENVIRONMENT_ALIAS = /* GraphQL */ `
+    mutation createEnvironmentAlias($data: CmsEnvironmentAliasInput!){
+        cms {
+            environmentAlias: createEnvironmentAlias(data: $data) {
+                data {
+                    ${fields}
+                }
+                error {
+                    code
+                    message
+                    data
+                }
+            }
+        }
+    }
+`;
+
+describe("Environment Alias test", () => {
+    const database = new Database();
+    const { invoke } = useGqlHandler({ database });
+    const initialEnvironment = { id: mdbid() };
+
+    beforeAll(async () => {
+        await database.collection("CmsEnvironment").insert({
```

Comment:
Please check how we've created the test environment in this file (and use the same approach):
`packages/api-headless-cms/__tests__/unlockingFields.test.js:13`